linux/map.jinja

```diff
@@ -185,3 +185,28 @@
          },
     },
 }, grain='os_family', merge=salt['pillar.get']('linux:storage')) %}
+
+
+{% set monitoring = salt['grains.filter_by']({
```

Comment:
You should decide if you want to have defaults in map.jinja or use `.get()` in support metadata, it's not needed to have both.